CMakeLists.txt

```diff
@@ -46,7 +46,6 @@ project(${PROJECT_NAME} ${PROJECT_ARGS})
 # Dependencies
 add_required_dependency("omniORB4 >= 4.1.4")
 add_required_dependency("omniDynamic4 >= 4.1.4")
-findpython()
```

Comment:
Where is it called ?
I have a build error because of this.

Reply:
You mean a configure error, right ? What is it ?
This is working in the CI, and @florent-lamiraux also got an error about this when he maybe hadn't a super clean workspace.

This is called in eigenpyConfig.cmake

Reply:
But you can install HPP without eigenpy.

In my setup, I have pinocchio and its dependency installed with robotpkg and HPP installed from source. I don't think `eigenpy` is cmake-included in this setup.